library/test/src/formatters/pretty.rs

```diff
@@ -45,8 +45,12 @@ impl<T: Write> PrettyFormatter<T> {
         self.write_short_result("FAILED", term::color::RED)
     }
 
-    pub fn write_ignored(&mut self) -> io::Result<()> {
-        self.write_short_result("ignored", term::color::YELLOW)
+    pub fn write_ignored(&mut self, may_message: Option<&'static str>) -> io::Result<()> {
+        if let Some(message) = may_message {
```

Comment:
What does 'may' stand for? For 'maybe' I suppose? The `Option` type already makes it clear that the message is optional. There's no need to repeat that information in the name.
```suggestion
    pub fn write_ignored(&mut self, message: Option<&'static str>) -> io::Result<()> {
        if let Some(message) = message {
```